src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestBigFilesIT.java

```diff
@@ -69,7 +69,7 @@ public void testManyRowsMultipleChannelsToMultipleTable()
 
     int numTables = 2;
     int numChannels = 4; // channels are assigned round-robin to tables.
-    int batchSize = 20000;
+    int batchSize = 10000;
```

Comment:
Reducing the size seems to go against the original purpose of this test, should we just disable the test while we're investigating?

Reply:
^ We don't have a limit on num rows coming in per channel, so the alerts probably expose a bug on ingest sdk where we are creating too big chunks that needs to be fixed, rather than changing this test.

Reply:
Instead of reducing the number of rows, we could make the payload values non-random, which would improve the compression ration and hence not produce this alert.

Reply:
Discussed offline that is is better to have a test with random values. We want to still produce large BDEC files (as the name of the test suggests), but a bit smaller than the server-side limit on compressed size. I can decrease the batchSize from 20k to 15k (instead of 10k).